josue/js/app.js

```diff
@@ -0,0 +1,52 @@
+import FormValidator from './Formvalidator.js';
+
+const form = document.querySelector('.form');
+const signInBtn = document.querySelector("#signBtn");
+const registerBtn = document.querySelector("#registerBtn");
+const submitBtn = document.querySelector("#submit");
+const registerFields = document.querySelector("#register");
+let fields = ['name','phone'];
+
+registerFields.remove();
+signInBtn.addEventListener("click",e=>{
+    registerFields.classList.add("register--hidden");
+    registerFields.remove();
+    submitBtn.innerText = "Sign In"
+    fields = ['name','phone']
+    validator.updateFields(fields,"signin")
+
+})
+registerBtn.addEventListener("click",e=>{
+    form.insertBefore(registerFields,submitBtn);
+    registerFields.classList.remove("register--hidden");
```

Comment:
can you add a better name to the class register? in here I don't know if you are hiding an input, a form or a button